### PR TITLE
Update deploy-a-jekyll-site.md for JEKYLL_ENV environment variable

### PR DESCRIPTION
**Pull request by dotZak**
`JEKYLL_ENV` can be set in two ways: as an environment variable, or as part of the build command. Since the build command is generic to both the _Production_ and _Preview_ environments, an environment variable should be used.

`JEKYLL_ENV` is important to the build process and must be set to `production` for certain features to work properly in Jekyll plugins and themes (the default is `development`, per [Jekyll's documentation on environments](https://jekyllrb.com/docs/configuration/environments/)), so the documentation for Cloudflare Pages should specify how to set it.

I've followed the formatting of the section above it as it is relevant. This should be placed in the documentation after the _Build configuration_ section (as it's part of the same process) and before _first deploy_ section (as it's required to be set before deployments) so that it's in the order of operations of the setup process.

### CODE REVIEW

File: products/pages/src/content/how-to/deploy-a-jekyll-site.md [hunk 1]
```diff
@@ -69,6 +69,16 @@ You can deploy your site to Cloudflare Pages by going to the dashboard, and crea
 
 </TableLayout>
 
+By default, Jekyll builds with `JEKYLL_ENV` set to `development`. For some features to work (e.g. the `google_analytics` feature of the default _minima_ theme) it must be set to `production`. To learn more, read [Jekyll Environment documentation](https://jekyllrb.com/docs/configuration/environments/). In Cloudflare Pages, select your GitHub repository that you created above then go to _Settings_ and select _Environment variables_. For the _Production_ environment, press the _Add Variables_ button, then provide the following information:
+
```
Comment (rita3ko) [inline products/pages/src/content/how-to/deploy-a-jekyll-site.md]: `then provide the following information` -> `then add the following environment variable`

**Review by rita3ko — CHANGES_REQUESTED** [review]
Thanks for the PR @dotZak ! Left a minor syntax comment for improvement.  -Rita

File: products/pages/src/content/how-to/deploy-a-jekyll-site.md (hunk 1 above)
Comment (dotZak) [reply products/pages/src/content/how-to/deploy-a-jekyll-site.md]: This has been addressed.

**Comment by dotZak** [comment]
@rita3ko Thanks for the feedback. I've updated the commit with the requested update.

**Review by kristianfreeman — APPROVED** [review]

**Comment by kristianfreeman** [comment]
@dotZak can you rebase this PR so we can merge it? thanks!

**Comment by kristianfreeman** [comment]
@dotZak it looks like the rebase here got a little messed up, do you mind opening a new PR and basing it off of our `production` branch? thanks!

**Pull request closed without merging**